ase/rtl/platform.vh

```diff
@@ -124,34 +124,32 @@
 `define LAT_UNDEFINED              300
 
 `define RDWR_VL_LATRANGE           20,118
-`define RDWR_VH_LATRANGE           240,270
+`define RDWR_VH_LATRANGE           140,180
 
 `define ASE_MAX_LATENCY            300
 
 
 /*
- * Disable Transaction shuffling
- * - In integrated mode, OutOfOrder channel control is used
- * - In discrete mode, InOrder channel control is used
+ * Select transaction shuffling mode.
  *
  * Infinite bandwidth test mode (independent of platform selection)
  * `define INFINITE_BANDWIDTH_MODE
- *
- * Specific platform features like interrupts and UMsgs are explicitly
+ */
+`ifdef INFINITE_BANDWIDTH_MODE
+ `define FORWARDING_CHANNEL            inorder_wrf_channel
```

Comment:
Is there a reason to keep inorder_wrf_channel and INFINITE_BANDWIDTH mode?

Reply:
Yes! If you are debugging looking for pipeline hazards in an AFU it is sometimes valuable to have the simulated FIU offer infinite bandwidth, leaving any hazards in the AFU exposed. All pipeline bubbles in infinite bandwidth mode are caused by the AFU.